database/seeds/UserTableSeeder.php

```diff
@@ -34,18 +34,11 @@ public function run()
     {
         // Add Admin Users
         $admins = [
```

Comment:
I actually think you should be able to remove all this work adding admin users now. so line 36-41 isn't needed since admins will be automatically created when they authenticate.